dll/win32/bcrypt/bcrypt_main.c

```diff
@@ -278,7 +279,8 @@ static const struct {
     /* ALG_ID_SHA1   */ { 20, BCRYPT_SHA1_ALGORITHM },
     /* ALG_ID_SHA256 */ { 32, BCRYPT_SHA256_ALGORITHM },
     /* ALG_ID_SHA384 */ { 48, BCRYPT_SHA384_ALGORITHM },
-    /* ALG_ID_SHA512 */ { 64, BCRYPT_SHA512_ALGORITHM }
+    /* ALG_ID_SHA512 */ { 64, BCRYPT_SHA512_ALGORITHM },
+    /* ALG_ID_ECDSA_P256 */ { 0, BCRYPT_ECDSA_P256_ALGORITHM },
```

Comment:
There's no need for specifying any hash length here?

Reply:
It should be 32 as of Sha 256

Reply:
Wine has `0`, because it is not a hash algo.

Reply:
yeah its the same what wine does, and i does not harm. i would prefer to keep it as is, otherwise syncing to wine needs additional effort, to clarify why it is different in reactos

Reply:
Ok, I must admit I am not really into eliptic curve algorythms